setup.py

```diff
@@ -1,8 +1,13 @@
+from os import sys, path
+
 from skbuild import setup
 
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from itkVersion import VERSION
```

Comment:
Instead of having `itkVersion` as a top-level module, it could be added within the itk package.

If not, we would have to list it in `py_modules` similarly to what is done in https://github.com/InsightSoftwareConsortium/ITKPythonPackage/pull/7/commits/86789704891617fed7510b87ac06d20f77fec936